angularSTEP/src/app/navbar/navbar.component.scss

```diff
@@ -0,0 +1,22 @@
+.spacer{
+    flex:     1 1 auto;
+}
+
+a:hover{
+    text-decoration: none;
+    color: red;
+}
+
+a:active{
+    text-decoration: none;
+    color: red;
+}
+
+a:visited{
+    text-decoration: none;
+    color: red;
+}
```

Comment:
With buttons, you will need to change this too. Is there no material preset to do this for you?